schemas/blocks/list_repeating_block.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$id": "/blocks/list_repeating_block.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/identifier"
+      },
+      "page_title": {
+        "$ref": "../common_definitions.json#/non_empty_string"
+      },
+      "type": {
+        "type": "string",
+        "enum": ["ListRepeatingQuestion"]
+      },
+      "question": {
```

Comment:
Are we supposed to support all kinds of questions (types)?

Reply:
Or do we need a separate definition for repeating block question?

Reply:
As far as I'm aware it's any question, as the limitations on the question weren't specified, and Mebin was fine with this impl of any question

Reply:
Can you make sure and ask Donna if it's all types of questions?

Reply:
Sure, this will be blocked until Monday though as she's on leave

Reply:
Donna cleared this - these question types are allowed 👍